source/_docs/manual-import.md

```diff
@@ -56,7 +56,7 @@ As long as you've chosen the same codebase (Drupal 7, Commerce Kickstart, etc.)
  git remote add pantheon <ssh_url>
  ```
 
-5. **Drupal only**: Move your settings file to `settings.local.php` so that it will be ignored by Git and included from Pantheon's `settings.php`. First, make sure that you can modify it, and restore the protections after the move:
+5. **Drupal only**: Move your settings file to `settings.local.php` and add it to `.gitignore` so that it will be ignored by Git and included from Pantheon's `settings.php`. Make sure that you can modify it, and restore the protections after the move:

```

Comment:
If we keep the step, we should add a snippet for D7 sites demonstrating how to include `settings.local.php`. If the step is removed, [line 66](https://github.com/pantheon-systems/documentation/pull/1459/files#diff-c6680de49ab763dc970a1fb757a86135R66) should be preserved in [Configuring settings.php](https://pantheon.io/docs/settings-php)